js/script.js

```diff
@@ -0,0 +1,88 @@
+const app = document.querySelector('#app');
+
+const headerText = document.createElement('h1');
+headerText.classList.add('leading-text');
+headerText.innerText = 'Awesome Books';
+app.appendChild(headerText);
+
+const noBookExistMessage = document.createElement('h3');
+noBookExistMessage.classList.add('no-book');
+noBookExistMessage.innerText = 'No Books are available';
+noBookExistMessage.style.display = 'none';
+app.appendChild(noBookExistMessage);
+
+let books = [];
+
+function getData() {
+  const data = JSON.parse(localStorage.getItem('data'));
+
+  if (data) {
+    books = data;
+  }
+}
+getData();
+
+const bookList = document.createElement('ul');
+bookList.classList.add('books');
+
+function displayBooks() {
+  bookList.innerHTML = '';
+  if (books.length) {
+    books.forEach((book) => {
+      const list = document.createElement('li');
+      list.classList.add('book');
+      list.innerHTML = `
+        <p class='book-title'> ${book.title} </p>
+        <p class='book-author'> ${book.author} </p>
+        <button class='btn remove' onClick="removeBook(${book.id})"> Remove </button>
+        <hr class='separator'>
+      `;
+      bookList.appendChild(list);
+    });
+  } else {
+    noBookExistMessage.style.display = 'block';
+  }
+}
+
+// eslint-disable-next-line no-unused-vars
+function removeBook(id) {
+  books = books.filter((book) => book.id !== id);
+  localStorage.setItem('data', JSON.stringify(books));
+  displayBooks();
+}
+
+displayBooks();
+app.appendChild(bookList);
+
+const addBookForm = document.createElement('form');
+addBookForm.setAttribute('id', 'add-book-form');
+addBookForm.innerHTML = `
+  <div>
+    <input type="text" name="title" id="title" required placeholder='Enter Book Title'/>
+  </div>
+  <div>
+    <input type="text" name="author" id="author" required placeholder='Enter Book Author'/>
+  </div>
+  <input type="submit" value="Add" class='btn submit'>
+`;
+app.appendChild(addBookForm);
+
+function CreateBookObject(title, author) {
+  this.title = title;
+  this.author = author;
+  this.id = books.length + 1;
+}
+
+addBookForm.addEventListener('submit', (event) => {
+  event.preventDefault();
+  const title = addBookForm.title.value;
+  const author = addBookForm.author.value;
+
+  const newBookObject = new CreateBookObject(title, author);
+  books.push(newBookObject);
+  noBookExistMessage.style.display = 'none';
+  displayBooks();
+  localStorage.setItem('data', JSON.stringify(books));
+  addBookForm.title.value = '';
+  addBookForm.author.value = '';
```

Comment:
- Kindly make sure that you are creating a function to add a book. This is a project requirement so you should create a function and call it to add a book. :+1: :smiley: :smiley: 

![image](https://user-images.githubusercontent.com/98363075/204396148-64703eba-0125-41d9-a3ee-e1647e28bd33.png)

- [OPTIONAL] I think it is a good idea to use indexes for the books however, kindly make sure that the indexes don't repeat when you remove a book and then you a new one. :eyes: :eyes: 

![image](https://user-images.githubusercontent.com/98363075/204396588-b61a5209-3a94-468a-a6f1-6e598d0e4c55.png)